src/workerd/io/docker-api.capnp

```diff
@@ -0,0 +1,292 @@
+
+# Copyright (c) 2017-2022 Cloudflare, Inc.
+# Licensed under the Apache 2.0 license found in the LICENSE file or at:
+#     https://opensource.org/licenses/Apache-2.0
+
+
+@0xfafd0bdf57acc528;
+using Cxx = import "/capnp/c++.capnp";
+using Json = import "/capnp/compat/json.capnp";
+
+$Cxx.namespace("workerd::docker_api");
+$Cxx.allowCancellation;
+
+
+struct Docker {
```

Comment:
It's best to avoid use of the trademarked name in this. I'd recommend `s/Docker/Container` and rename this file to `container-api.capnp`

Reply:
This is literally docker's api, there's no getting around that.

Reply:
Well, this part is in our capnp files :-)